### DISCUSSION TITLE
Remove unwanted object from pictures

### DISCUSSION THREAD

**Discussion by manuelkamp** [mudler/LocalAI#1516] [General]
I have running my own LocalAI instance for a while now. I really got into using LLMs for daily things. Now I found a site (https://www.photoroom.com/tools/remove-object-from-photo) where you can mark part of a picture and it removes it. I am curious how this could be done, since I did not really looked into stablediffusion things yet (I just tried some different models just for fun with LocalAI). Ideally I would look for some kind of frontend (like AnythingLLM) for all stablediffusion things like image generating and manipulating like this 'remove objects from pictures' thing.

Sorry if that is OT, but I figured out, since I want to use my LocalAI for that, maybe anyone has some experience here already?

**Comment by moritztim** [comment]
I recommend [Fooocus](https://github.com/lllyasviel/Fooocus) for SDXL (latest, biggest and best version of stable diffusion) or [Stable Diffusion Web UI](https://github.com/AUTOMATIC1111/stable-diffusion-webui) for previous versions (I think it also works with SDXL but it's more complicated to use than Foocus).
Both of these allow you to use inpainting to mask out parts of an image and add, remove or modify objects in the masked area.

**Reply by manuelkamp** [reply]
thank you, both solutions seem to use their own backend for image generation and not implement LocalAI, correct? That may be a bit of a hickup, since my server only has one GPU and I can only pass it through to one LXC (either LocalAI or the other), so what I search for, would be a frontend like Fooocus, which uses LocalAI for image generation?

**Reply by moritztim** [reply]
Yes they do. I was hoping that you can somehow use your own backend for them but you'd have to dig into the docs to figure that out. Idk any other frontends

**Comment by moritztim** [comment]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/2260
This might help
